### PR TITLE
feat: avatar

### PR DESCRIPTION
**Pull request by mikesch803**
adding avatar with text and image with 3 different size

### CODE REVIEW

**Comment by mikesch803** [comment]
here is the link https://wonderful-cori-662c94.netlify.app/

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    border-radius: 50%;
+    object-fit: cover;
+}
+
+.avatar-txt {
+    background-color: var(--color-primary);
+}
+
+.avatar-ls {
+width: 5rem;
```
Comment (nayyyhaa) [inline components/avatar.css]: Can make the overall file indentation consistent.

**Review by nayyyhaa — COMMENTED** [review]
Overall code looks good. Only indentation part could be improved :)

File: components/avatar.css (hunk 1 above)
Comment (mikesch803) [reply components/avatar.css]: thanks, i made the changes

**Pull request closed without merging**